themes/kairos-theme/LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 YOUR_NAME_HERE
+Copyright (c) 2017 Kairos Website Theme
```

Comment:
Any way to make this dynamic? I guess it's too much trouble, but we shouldn't forget to change this later I guess.